docs/store/datastore.md

```diff
@@ -115,4 +116,12 @@ be used if this file is located in a location on shared storage, accessible to p
 * `GCP_CREDENTIALS` &mdash; when the credentials file cannot be mounted to the pod, this secret or environment variable
 may contain the contents of this file. If configured in the function pod, MLRun dumps its contents to a temporary file 
 and points `GOOGLE_APPLICATION_CREDENTIALS` at it. An exception is `BigQuerySource`, which passes `GCP_CREDENTIALS`'s
-contents directly to the query engine.
\ No newline at end of file
+contents directly to the query engine.
+
+### Databricks file system
+```{Admonition} Note
+DBFS does not support Spark, http, or memory.
```

Comment:
I find this note very confusing. First of all, it's not that DBFS does not support Spark, it's that Spark as the processing engine does not support `dbfs://` paths. Also, I'm not sure how `http` and `memory` come into play here - @assaf758, can you clarify this statement? What do you refer to?

Reply:
well, this section of the doc is talking here about datastore, so in that regard I said dbfs datastore does not support spark ingestion engine. But it's semantic - we mean the same :-)
As to http and memory, the discussion was about datastores which have "caveats" of missing support, so I mentioned http and memory as two datastores [showing in our doc](https://docs.mlrun.org/en/latest/store/datastore.html#shared-data-stores) that spark ingestion do not support.
Since it's all the caveats are spark related, i'm fine with moving this to spark section.

Reply:
@assaf758 
I think it should be in both places. you never know from which side a user starts reading.
Spark - you mean https://docs.mlrun.org/en/latest/runtimes/spark-operator.html, correct?

Reply:
Yes. notice that there is also a "remote-spark" that does not have a description (the [link](https://docs.mlrun.org/en/latest/concepts/functions-overview.html#kinds-of-functions-runtimes) in the overview section points directly to feature-set ingestion example)

Reply:
all complete except an overview topic for remote-spark. Noted as AI